iosHackthonApp/Modules/Features/Login/Sources/Login/LoginViewModel.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  File.swift
```

Comment:
It's annoying that this happens now when you create a new file

Reply:
ohhh yeah..changed it to `LoginViewModel.swift`

Reply:
on YNAP project they just delete all of these heading comments - which is weird at first but they are pretty pointless I suppose 🤷